src/renderer/api/kube-api.ts

```diff
@@ -9,7 +9,9 @@ import { apiKube } from "./index";
 import { createKubeApiURL, parseKubeApi } from "./kube-api-parse";
 import { KubeJsonApi, KubeJsonApiData, KubeJsonApiDataList } from "./kube-json-api";
 import { IKubeObjectConstructor, KubeObject } from "./kube-object";
-import { kubeWatchApi } from "./kube-watch-api";
+import byline from "byline";
+import { ReadableWebToNodeStream } from "readable-web-to-node-stream";
```

Comment:
We can probably write our own helpers for this (or port logic from master branch). Biggest issue I see here is that we are using `fetch` which does not return "nodejs" compatible stream. `byline` wants nodejs stream, `ReadableWebToNodeStream` converts browser stream to nodejs stream ... it's a hack.

Reply:
Btw, since in `renderer` we have enabled `nodeIntegration: true` could we use native [node-js stream-api](https://nodejs.org/dist/latest-v12.x/docs/api/stream.html) in both processes?

Reply:
We could use native but `fetch` does not return nodejs native streams. That's why I'm converting streams here. Probably story for an another PR (we could probably replace fetch with nodejs library, like `got`)

Reply:
I meant if we could throw away `fetch` and use native node.js stream for both `main` and `renderer` => no transform stream(s) would be needed client-side.

Reply:
> I meant if we could throw away `fetch` and use native node.js stream for both `main` and `renderer` => no transform stream(s) would be needed at the client-side.

Yeah, sounds like a plan. Only downside would be that browser devtools would not show up requests if we use nodejs libraries... but that probably is acceptable tradeoff.